tests/python/cpu/test_mkldnn.py

```diff
@@ -0,0 +1,94 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
tests in ```tests/python/cpu``` are currently not run in CI. Please add it to the MKLDNN jobs.

Reply:
@marcoabreu This PR only adds to existing test, no new test file is created. Can you be specific which CI jobs you are referring to ? If there is a gap in CI, a new JIRA item can be created to address it.

Reply:
According to https://github.com/apache/incubator-mxnet/blob/master/Jenkinsfile#L439, which in the end points to https://github.com/apache/incubator-mxnet/blob/master/ci/docker/runtime_functions.sh#L387, we're not running the tests located in tests/python/cpu. Please rename tests/python/cpu to tests/python/mkl and add another function in the same style as https://github.com/apache/incubator-mxnet/blob/master/Jenkinsfile#L458.